primer/brand
# Add `instagram` to `MinimalFooterSocialLinks`

## Summary

This adds `instagram` to the `MinimalFooterSocialLinks`.

## List of notable changes:

- **added** `instagram` to `MinimalFooterSocialLinks` to avoid Type error

## Steps to test:

When adding `instagram` like

```ts
<MinimalFooter socialLinks={['twitter', 'facebook', 'youtube', 'linkedin', 'github', 'tiktok', 'instagram']}>
```

it works, and the icon is rendered, but there is a Type error that doesn't pass [our CI check](https://github.com/github/githubuniverse.com/actions/runs/5923516310/job/16059302179?pr=1797):

```bash
Type error: Type '"instagram"' is not assignable to type '"twitter" | "github" | "linkedin" | "youtube" | "facebook" | "twitch" | "tiktok"'.

  4 | export const Footer = () => {
  5 |   return (
> 6 |     <MinimalFooter socialLinks={['twitter', 'facebook', 'youtube', 'linkedin', 'github', 'tiktok', 'instagram']}>
    |                                                                                                    ^
```

## Supporting resources (related issues, external links, etc):

- This came up in https://github.com/github/githubuniverse.com/pull/1797#issuecomment-1685914785

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> Please try to provide before and after screenshots or videos

N/A
